raiden/tests/unit/transfer/mediated_transfer/test_mediation_fee.py

```diff
@@ -61,34 +61,52 @@ def test_basic_fee():
 
 
 def test_imbalance_penalty():
+    r""" Test an imbalance penalty by moving back and forth
```

Comment:
Why the raw string here? Something to do with the plain text graph.

Reply:
I didn't want to escape the backslashes in the ascii diagram because that would mess up the diagram.